data/mm/world/overworld.yml

```diff
@@ -222,12 +222,14 @@
   exit:
     "Swamp": "true"
   locations:
-    "Tourist Information Pictobox": "can_rescue_koume"
-    "Tourist Information Boat Cruise": "can_rescue_koume && event(CLEAN_SWAMP) && has(BOW)"
+    "Tourist Information Pictobox": "event(KOUME) || has(PICTOGRAPH_BOX) || (has(MASK_DEKU) && (has(BOW) || has(HOOKSHOT) || has(MASK_ZORA))"
```

Comment:
This is the pictobox check itself (that is, koume saying thank you and giving you an item), so this seems wrong ; should probably just be event(KOUME)